base/serialize.jl

```diff
@@ -116,16 +114,16 @@ function serialize(s::SerializationState, t::Tuple)
         writetag(s.io, LONGTUPLE_TAG)
         write(s.io, Int32(l))
     end
-    for i = 1:l
-        serialize(s, t[i])
+    for e in t
```

Comment:
Good point, I haven't thought about it. Would you suggest another name which is normally used for elements of a vector in Julia core? (I think I saw `x` a couple of times)

Since this PR is already merged, I'm going to include the change into the next one.